src/gradletestrunnersmoketest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+gradletestrunnersmoketest () {
+  export TEST_RUNNER_SERVER_SPECIFICATION_FILE="${NAMESPACEINUSE}.json"
+  cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-client
+  ../gradlew clean assemble  
+  cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests
+  export ORG_GRADLE_PROJECT_datarepoclientjar=$(ls ../datarepo-client/build/libs/*jar)
+  ./gradlew runTest --args="suites/PRSmokeTests.json tmp/TestRunnerResults"
```

Comment:
Nit: Could you add some logging statements here to help debug failures? (e.g. echo "Building Data Repo client library", echo "Running TestRunner suite")

Reply:
Yes, good idea!